### ISSUE TITLE
accessor: support tags

### ISSUE THREAD

**Issue by iksaif** [criteo/biggraphite#95]
- Start by optionally supporting tags in the accessor
- Then implement it for cassandra.

**Comment by dpanth3r** [comment]
Can we be a bit more descriptive? Do we mean tags in a way similar to Prometheus?

**Comment by iksaif** [comment]
Yes, labels like prometheus. I haven't had time to really think about it but the idea would be able to support at least what prometheus needs: foo.bar{label1="foo",label2"="test"}

**Comment by Petee87** [comment]
Do you plan to add original HTTP tagdb support to bg carbon cache plugin as tagdb option?

**Comment by iksaif** [comment]
I think you should already be able to use the original HTTP TagDB plugin, I never tried though.

**Comment by Petee87** [comment]
The http tagdb not working for me. I think the bg carbon overwrite.
biggraphite - 0.14.2, graphite(web,carbon) - 1.1.2
